rmf_task/test/integration/test_backups.cpp

```diff
@@ -15,24 +15,64 @@
  *
 */
 
+#include <iostream>
+#include <filesystem>
 #include <rmf_task/BackupFileManager.hpp>
 
 #include "../mock/MockDelivery.hpp"
 
 #include <rmf_utils/catch.hpp>
 
+const std::string backup_root_dir = "/tmp/rmf_task/test_backups";
```

Comment:
Since this variable is used as a path, I recommend declaring it as a path, i.e.:
```
const std::filesystem::path backup_root_dir = "/tmp/rmf_task/test_backups";
```

Then when you use it down below, instead of doing ` + "/...` you can do ` / "...`, e.g.:
```
backup_root_dir / "group"
```
instead of
```
backup_root_dir + "/group"
```

This is the new standard platform-agnostic way to construct paths.